Description: Disable tests failing with Debian's PowerMockito
 The Maven dependency chain only uses mockitor-core, whereas
 Debian's libpowermock-java uses the full mockito distribution.
 Mockito in turn relies on internal behavior in cglib which
 has changed and thus no longer works as intended. See bug
 #707298 for details.
Author: Ingo Bauersachs <ingo@jitsi.org>
Last-Update: 2016-11-24
diff --git a/src/test/java/org/jitsi/dnssec/TestPriming.java b/src/test/java/org/jitsi/dnssec/TestPriming.java
index f946eb4..5eff4ba 100644
--- a/src/test/java/org/jitsi/dnssec/TestPriming.java
+++ b/src/test/java/org/jitsi/dnssec/TestPriming.java
@@ -21,6 +21,7 @@ import static org.powermock.api.mockito.PowerMockito.whenNew;
 
 import java.io.IOException;
 
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.powermock.core.classloader.annotations.PrepareForTest;
@@ -81,6 +82,7 @@ public class TestPriming extends TestBase {
 
     @Test
     @PrepareMocks("prepareTestDnskeyPrimeResponseWithMismatchedFootprintIsBad")
+    @Ignore
     public void testDnskeyPrimeResponseWithMismatchedFootprintIsBad() throws Exception {
         Message response = resolver.send(createMessage("www.ingotronic.ch./A"));
         assertFalse("AD flag must not be set", response.getHeader().getFlag(Flags.AD));
@@ -96,6 +98,7 @@ public class TestPriming extends TestBase {
 
     @Test
     @PrepareMocks("prepareTestDnskeyPrimeResponseWithMismatchedAlgorithmIsBad")
+    @Ignore
     public void testDnskeyPrimeResponseWithMismatchedAlgorithmIsBad() throws IOException, NumberFormatException, DNSSECException {
         Message response = resolver.send(createMessage("www.ingotronic.ch./A"));
         assertFalse("AD flag must not be set", response.getHeader().getFlag(Flags.AD));
diff --git a/src/test/java/org/jitsi/dnssec/validator/TestNsec3ValUtils.java b/src/test/java/org/jitsi/dnssec/validator/TestNsec3ValUtils.java
index 43e4669..93a8187 100644
--- a/src/test/java/org/jitsi/dnssec/validator/TestNsec3ValUtils.java
+++ b/src/test/java/org/jitsi/dnssec/validator/TestNsec3ValUtils.java
@@ -27,6 +27,7 @@ import org.jitsi.dnssec.AlwaysOffline;
 import org.jitsi.dnssec.PrepareMocks;
 import org.jitsi.dnssec.TestBase;
 import org.junit.Assume;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.invocation.InvocationOnMock;
@@ -134,6 +135,7 @@ public class TestNsec3ValUtils extends TestBase {
 
     @Test
     @PrepareMocks("prepareTestPublicKeyLoadingException")
+    @Ignore
     public void testPublicKeyLoadingException() throws IOException {
         Message response = resolver.send(createMessage("www.wc.nsec3.ingotronic.ch./A"));
         assertFalse("AD flag must not be set", response.getHeader().getFlag(Flags.AD));
